src/test/java/org/gaul/s3proxy/EncryptedBlobStoreTest.java

```diff
@@ -0,0 +1,153 @@
+package org.gaul.s3proxy;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.ByteSource;
+import com.google.common.io.ByteStreams;
+import com.google.common.net.MediaType;
+import com.google.inject.Module;
+import org.assertj.core.api.Fail;
+import org.jclouds.ContextBuilder;
+import org.jclouds.blobstore.BlobStore;
+import org.jclouds.blobstore.BlobStoreContext;
+import org.jclouds.blobstore.domain.Blob;
+import org.jclouds.blobstore.domain.BlobMetadata;
+import org.jclouds.blobstore.domain.PageSet;
+import org.jclouds.blobstore.domain.StorageMetadata;
+import org.jclouds.blobstore.options.PutOptions;
+import org.jclouds.io.ContentMetadata;
+import org.jclouds.logging.slf4j.config.SLF4JLoggingModule;
+import org.junit.After;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.gaul.s3proxy.S3ProxyConstants.*;
+
+public final class EncryptedBlobStoreTest {
+    private static final int BYTE_SOURCE_SIZE = 1024;
+    private static final ByteSource BYTE_SOURCE = TestUtils.randomByteSource()
+        .slice(0, BYTE_SOURCE_SIZE);
+    private BlobStoreContext context;
+    private BlobStore blobStore;
+    private String containerName;
+    private BlobStore encryptedBlobStore;
+    private Properties properties;
+
+    @Before
+    public void setUp() throws Exception {
+        containerName = TestUtils.createRandomContainerName();
+
+        context = ContextBuilder
+                .newBuilder("transient")
+                .credentials("identity", "credential")
+                .modules(ImmutableList.<Module>of(new SLF4JLoggingModule()))
+                .build(BlobStoreContext.class);
+        blobStore = context.getBlobStore();
+        blobStore.createContainerInLocation(null, containerName);
+
+        properties = new Properties();
+        properties.put(PROPERTY_ENCRYPTED_BLOBSTORE, "true");
+        properties.put(PROPERTY_PEACEMAKR_API_KEY, "");
```

Comment:
should just work?

Reply:
could org test api key

Reply:
ie, https://api.peacemakr.io/api/v1/org/key/test

Reply:
We no longer need to provide an api key to run local-test only.